docs/proposals/lifelong-learning/unstructured-lifelong-learning.md

```diff
@@ -0,0 +1,247 @@
+* [Unstructured Lifelong Learning](#lifelong-learning)
+   * [Motivation](#motivation)
+     * [Goals](#goals)
+     * [Non\-goals](#non-goals)
+   * [Proposal](#proposal)
+     * [Use Cases](#use-cases)
+   * [Design Details](#design-details)
+     * [CRD API Group and Version](#crd-api-group-and-version)
+     * [Lifelong learning CRD](#lifelong-learning-crd)
+     * [Lifelong learning type definition](#lifelong-learning-job-type-definition)
+     * [Lifelong learning sample](#lifelong-learning-job-sample)
+     * [Validation](#validation)
+   * [Controller Design](#controller-design)
+     * [Lifelong Learning Controller](#lifelong-learning-controller)
+     * [Downstream Controller](#downstream-controller)
+     * [Upstream Controller](#upstream-controller)
+     * [Details of api between GM(cloud) and LC(edge)](#details-of-api-between-gmcloud-and-lcedge)
+   * [Workers Communication](#workers-communication)
+
+# Unstructured Lifelong Learning
+## Motivation
+
+Edge cloud synergy lifelong learning has been proposed and realized through Sedna in [ATCII example](https://github.com/kubeedge/sedna/blob/main/examples/lifelong_learning/atcii/README.md) which handles structured data. However, the last version of Sedna lifelong learning scheme has the following limitations.
+
+* **It can't directly apply to unstructured data scenarios** such as images, texts, voices, etc, which instead a majority of AI applications involve. The adaption to unstructured data is essential for Sedna lifelong learning to run more diversified applications.
+
+* **It hasn't enable to detect or tackle unseen samples**, also called heterogeneous samples, at the inference stage. As is well known, Non-IID is prevailing situation that exists in distributed AI. When encountering heterogeneous data, the real-time inference performance of original model is greatly reduced which might cause unacceptable loss in certain scenarios. Hence, how to detect heterogeneous or unseen data in advance and process them properly in real time become sigificant topics.
+
+* **It has not realized efficient unseen task processing**. As unseen samples are collected, turning them to be seen samples in order to make model smarter is challenging and important. Efficiently training unseen sample training in addition to retraining all the data is worth studying. 
+
+In this version of lifelong learning, we seek to fundamentally overcome the above challenges of edge-cloud distributed machine learning.
+### Goals
+
+* Supports unstructured lifelong learning.
```

Comment:
It is not clear for reader to understand, like "the goal of unstructured lifelong learning is to support unstructured lifelong learning". I suggest to give a concrete explain about this goal, at least you can say "Support unstructured dataset like images"

Reply:
done